schemas/oic.collection.linkslist-schema.json

```diff
@@ -7,10 +7,5 @@
 		       "$ref": "oic.collection-schema.json#/definitions/oic.collection.alllinks"
     }
   },
-  "type": "object",
-  "properties": {
-    "links":  {
-      "$ref": "#/definitions/oic.collection.alllinks"
-    }
-  }
+  "$ref": "#/definitions/oic.collection.alllinks"
```

Comment:
Yes.  This now an array of links and so this schema just $refs to alllinks in the collections schema that is just that; an array of links.  The schema no longer represents a JSON object.